gradle-plugin/it/src/it/dockerfile-simple/expected/openshift.yml

```diff
@@ -71,12 +71,10 @@ items:
     - type: ConfigChange
     - imageChangeParams:
         automatic: true
-        containerNames:
-        - gradle-dockerfile-simple
         from:
           kind: ImageStreamTag
           name: dockerfile-simple:latest
-          namespace: gradle
+          namespace: "@ignore@"
```

Comment:
https://github.com/eclipse/jkube/pull/915/files#r709786504

Reply:
This gets evaluated to current namespace if you're logged into OpenShift cluster. I didn't want test to fail in case you're logged into an openshift cluster. So I thought of ignoring this field.